provider/unit_test_data/resource_service/creation_data_provider.yaml

```diff
@@ -390,9 +394,8 @@
        "_key": "service_test_create_kpis",
        "description":"Terraform unit test",
        "enabled":0,
-       "entity_rules":[
-
-       ],
+       "entity_rules":[],
+       "is_healthscore_calculate_by_entity_enabled":1,
```

Comment:
ah, yes, thank you!

still want to come up with something smaller in data provider